packages/ui/src/components/popover/popover.tsx

```diff
@@ -0,0 +1,198 @@
+import {h, Component} from 'preact';
+import * as styles from './popover.scss';
+import {PopoverContent} from './popover-content';
+
+export enum PopoverVerticalPositions {
+  Top = 'top',
+  Bottom = 'bottom',
+}
+export enum PopoverHorizontalPositions {
+  Left = 'left',
+  Right = 'right',
+}
+export enum PopoverTriggerMode {
+  Click = 'click',
+  Hover = 'hover',
+}
+
+const CLOSE_ON_HOVER_DELAY = 500;
+
+const defaultProps = {
```

Comment:
general comment - wasn't able to position the popover at the side of the element. If it is a product req. please test it as well

Reply:
have you passed a valid properties into popover component according to Popover API?
If so, show me how it looked.

Reply:
@ainvoner  @semarche-kaltura can you resolve this one?

Reply:
Popover position works as expected.
One known limitation: if popover opens beyond the visible area of the player - the popover will be overlapped by player